test_runner/regress/test_tenant_detach.py

```diff
@@ -63,9 +63,7 @@ def test_tenant_reattach(
     # Attempts to connect from compute to pageserver while the tenant is
     # temporarily detached produces these errors in the pageserver log.
     env.pageserver.allowed_errors.append(f".*Tenant {tenant_id} not found.*")
-    env.pageserver.allowed_errors.append(
```

Comment:
Since #5711, QueryError::Shutdown is only logged at info level, so if everything is working nicely we should be able to entirely remove these allowed_errors lines for "Current state: Stopping", rather than replacing them

Reply:
Thanks for the info! @jcsp 
(That is the reason I didn't get any error in pytests without replacing these `Current State: Stopping` errors.)

I think it is better to entirely remove these replaces errors from testing.
What do you think? @jcsp